src/stores/account.ts

```diff
@@ -245,4 +250,23 @@ export default class AccountStore implements IAccountStore {
     this._privateKeyToRecovery = null;
     return account;
   };
+
+  fixKeyLength = async (v: ProtectedPrivateKey, passphrase: string) => {
+    const key = V3toRaw(await fs.promises.readFile(v.path, "utf8"), passphrase);
+    if (key.length < 32) {
+      fs.promises.writeFile(
+        v.path,
+        (
+          await new Wallet(
+            new SigningKey(
+              Buffer.concat([Buffer.alloc(32 - key.length), key], 32)
+            )
+          ).encrypt(passphrase)
+        )
+          .replace(/"id":"[0-9A-z-]*"/, `"id":"${v.keyId}"`)
+          .replace("C", "c")
+      );
+      key.fill(0);
```

Comment:
is this still needed? (because `key` variable doesn't seem to be used furthermore)

Reply:
This effectively wipes raw private key regardless of garbage collection time.
This was previously questioned that this might cause copy-on-write and does not modify physical data on memory,

But I checked Node.js and ECMAScript specification thoroughly, and concluded
Buffer - Uint8Array - [ArrayBuffer](https://tc39.es/ecma262/multipage/structured-data.html#sec-arraybuffer-objects) - [DataBlock](https://tc39.es/ecma262/multipage/ecmascript-data-types-and-values.html#sec-data-blocks) is a mutable data block, and doesn't cause copy-on-write implicitly.

So yep, I think it's needed.

Reply:
- I think that if some volatile process can look up the physical memory of another process, there're many things we should consider and... I guess it doesn't seem practical. [^1]
- Of course, it's not a serious trade-off, so cleaning up memory isn't necessarily something to be avoided here. But it seems necessary to leave a comment about it.

[^1]: The reason [Spectre](https://en.wikipedia.org/wiki/Spectre_(security_vulnerability)) is a serious security vulnerability is that it can refer to the memory of other processes, which means that it is an important assumption of modern software development that you cannot see it.

Reply:
Left a comment on two different case of zero-filling.